plugins/woocommerce/includes/admin/settings/class-wc-settings-page.php

```diff
@@ -31,6 +31,42 @@ abstract class WC_Settings_Page {
 		 */
 		public $icon = 'settings';
 
+		/**
+		 * Settings field types which are known.
+		 *
+		 * @var string
+		 */
+		protected $types = array(
```

Comment:
Nit: Should we use constants instead of strings? Example (from ChatGPT):
```php
class Settings {
    const TYPE_TITLE = 'title';
    const TYPE_INFO = 'info';
    // ... Other constants

    /**
     * Settings field types which are known.
     *
     * @var string[]
     */
    protected $types = [
        self::TYPE_TITLE,
        self::TYPE_INFO,
        // ... Use constants here
    ];
}
```

This approach avoids magic strings and makes makes child classes less error-prone